KukaApp/components/Header.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { StyleService, Text, useStyleSheet } from '@ui-kitten/components';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+const themedStyles = StyleService.create({
+  headerContainer: {
+    backgroundColor: 'color-primary-default',
+    fontFamily: 'OpenSans-Bold',
+    fontWeight: '700',
+    fontSize: 26,
+    paddingLeft: 18,
+    paddingBottom: 10,
+  },
+});
+
+export const Header = ({ text }) => {
```

Comment:
What is the difference between `<HeaderText>` and `<Header>`? (i.e. `components/Header` and `components/HeaderText`)

Reply:
Not much besides the safe area padding. There'll probably be another refactor between the two in the future but for now, I'm taking a wait-and-see attitude and focus more on getting the safe area changes in.